src/dataflow/src/render/reduce.rs

```diff
@@ -488,13 +418,14 @@ where
     })
 }
 
-// Render a single reduction tree that computes a set of mins and max aggregations
-// hierarchically. Note that because we know all the aggregations are mins and maxes
-// we can also ignore the distinct bit here.
-fn build_min_max<G>(
+// Render a single reduction tree that computes aggregations
+// hierarchically. If the input is monotonic, we further specialize and
+// render them as a fused series of monoids similar to the accumulable reductions.
```

Comment:
we still ignore the distinct bit here. is that fine or not? either way, should at least be documented in a comment